src/pages/product/link.tsx

```diff
@@ -0,0 +1,235 @@
+            <Text className={classes.copy}>
+              Models added to Vault will not be deleted even if the creator removed them from
+              Civitai.
+            </Text>
+          </Grid.Col>
+        </Grid>
+        <Grid gutter={40} gutterMd={80}>
+          <Grid.Col md={6}>
+            <Flex justify="center" className={classes.gradientContainer}>
+              <Image
+                src="/images/link/download.png"
+                alt="download"
+                id="download"
+                width="auto"
+                imageProps={{
+                  style: { objectFit: 'cover', objectPosition: 'top', height: '100%' },
+                }}
+              />
+              <div className={classes.gradientBox} />
+            </Flex>
+            <Title className={classes.heading} order={3}>
+              Manage files
+            </Title>
+            <Text className={classes.copy}>
+              Directly add or remove any models from Civitai to your Stable Diffusion instance.
+            </Text>
+          </Grid.Col>
+          <Grid.Col md={6}>
+            <Flex justify="center" className={classes.gradientContainer}>
+              <Image
+                src="/images/link/activity.png"
+                alt="activity"
+                id="activity"
+                width="auto"
+                imageProps={{
+                  style: { objectFit: 'cover', objectPosition: 'top', height: '100%' },
+                }}
+              />
+              <div className={classes.gradientBox} />
+            </Flex>
+            <Title className={classes.heading} order={3}>
+              Keep track of activities
+            </Title>
+            <Text className={classes.copy}>
+              See the history of all the models you have added to your Stable Diffusion instance.
+            </Text>
+          </Grid.Col>
+        </Grid>
+
+        {/* TODO: Add video once created */}
+        {/* <AspectRatio ratio={16 / 9} my={40}>
+        <YoutubeEmbed videoId="MaSRXvM05x4" />
+      </AspectRatio> */}
+
+        <Flex justify="center" w="100%">
+          <Button
+            variant="filled"
+            color="blue"
+            size="lg"
+            radius="xl"
+            my={40}
+            component={NextLink}
+            href={buttonData.href}
+            rel="nofollow noreferrer"
+          >
+            {buttonData.text}
+          </Button>
+        </Flex>
+      </Container>
+    </>
+  );
+}
+
+const useStyles = createStyles((theme) => ({
+  heroContainer: {
+    marginTop: 40,
+    marginBottom: 40,
+  },
+  heroTitle: {
+    fontSize: 32,
+    fontWeight: 700,
+    color: theme.colorScheme === 'dark' ? '#fff' : theme.colors.gray[9],
+    [containerQuery.largerThan('md')]: {
+      fontSize: 40,
+    },
+  },
+  heroText: {
+    fontSize: 14,
+    fontWeight: 500,
+  },
+  heading: {
+    fontSize: 24,
+    fontWeight: 700,
+    color: theme.colorScheme === 'dark' ? '#fff' : theme.colors.gray[9],
```

Comment:
The color `white` I suppose should be available in something like `theme.colors.common.white` just guessing, but better than hard-coding white and maybe check this https://bootcamp.uxdesign.cc/alternative-colors-for-pure-white-in-ui-ux-design-f861ab5a33d7

Reply:
Our designs utilize white for this. However its a good call out to use the `theme.white` for consistency. This happens to be an off white. Ill update our designer so we can get better consistency with our design system.